app/helpers/gov_uk_form_helper.rb

```diff
@@ -0,0 +1,54 @@
+module GovUkFormHelper
```

Comment:
👍 This is a great starting point to define abstractions for our forms.

Reply:
I'm glad you like it. I think helpers allow the code to be tidied up and remove logic from the view partials.